src/DocumentDB.ChangeFeedProcessor/Monitoring/HealthEventLevel.cs

```diff
@@ -0,0 +1,22 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+//----------------------------------------------------------------
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.Monitoring
+{
+    /// <summary>
+    /// The healthiness level
```

Comment:
The commend should say something like the level (or severity) of health event (or health issue).